PROTOCOL.md

```diff
@@ -695,6 +708,15 @@ For example, given the schema string (pretty printed for readability. The entire
 
 Writers should reject any transaction that contains data where the expression `x > 3` returns `false` or `null`.
 
+## Check Constraints
+
+- Check constraints are stored at the table-level in `Metadata.configuration` and consist of a `name` and an `expression`. An input row satisfies a constraint when `expression` evaluates to `true`.
+  - They are stored in `Metadata.configuration` with the key `delta.constraints.{name}`.
+  - `expression` is stored as the value of `delta.constraints.{name}` and SHOULD be parsed as a boolean SQL expression.
+- CHECK constraints are strongly enforced:
```

Comment:
Can we be consistent with our usage. So far we have
- [711] `## Check Constraints`
- [713] `Check constraints`
- [716] `CHECK constraints`